docs/src/neuralestimators_amorized.md

```diff
@@ -0,0 +1,261 @@
+    )
+    
+    # Final layer with parameter constraints
+    final_layer = Parallel(
+        vcat,
+        Dense(w, 1, x -> ν_min + (ν_max - ν_min) * σ(x)),     # ν1
+        Dense(w, 1, x -> ν_min + (ν_max - ν_min) * σ(x)),     # ν2
+        Dense(w, 1, x -> α_min + (α_max - α_min) * σ(x)),     # α
+        Dense(w, 1, x -> β_min + (β_max - β_min) * σ(x)),     # β
+        Dense(w, 1, x -> λ_min + (λ_max - λ_min) * σ(x)),     # λ
+        Dense(w, 1, x -> τ_min + (τ_max - τ_min) * σ(x))      # τ
+    )
+    
+    # Outer network - maps aggregated features to parameters
+    ϕ = Chain(
+        Dense(w, w, relu),
+        Dense(w, w, relu),
+        final_layer
+    )
+    
+    # Combine into a DeepSet
+    network = DeepSet(ψ, ϕ)
+    
+    # Initialize neural Bayes estimator
+    estimator = PointEstimator(network)
+    
+    return estimator
+end
+```
+
+## Training the Neural Estimator
+
+Neural estimators, like all deep learning methods, require a training phase where they learn the mapping from data to parameters. Here we will train the estimator, simulating data as we go where the sampler provides new parameter vectors from the prior, and a simulator can be provided to continuously simulate new data conditional on the parameters.
+
+```julia
+# Create the neural estimator
+estimator = create_neural_estimator()
+
+# Train with on-the-fly simulation
+trained_estimator = train(
+    estimator,
+    sample,              # Parameter sampler function
+    simulate,            # Data simulator function
+    m = 100,             # Number of trials per parameter vector
+    K = 1000,           # Number of training parameter vectors
+    K_val = 200,        # Number of validation parameter vectors
+    loss = Flux.mae,     # Mean absolute error loss
+    epochs = 5,         # Number of training epochs
+    epochs_per_Z_refresh = 1,  # Refresh data every epoch
+    epochs_per_θ_refresh = 5,  # Refresh parameters every 5 epochs
+    batchsize = 64,            # Batch size for training
+    verbose = true
+)
+```
+
+## Assessing Estimator Performance
+
+We can assess the performance of our trained estimator on held-out test data:
+
+```julia
+# Generate test data
+n_test = 100
+θ_test = sample(n_test)
+Z_test = simulate(θ_test, 100)
+
+# Assess the estimator
+parameter_names = ["ν1", "ν2", "α", "β", "λ", "τ"]
+assessment = assess(
+    trained_estimator, 
+    θ_test, 
+    Z_test; 
+    parameter_names = parameter_names
+)
+
+# Calculate performance metrics
+bias_results = bias(assessment)
+rmse_results = rmse(assessment)
+println("Bias: ", bias_results)
+println("RMSE: ", rmse_results)
+```
+
+## Visualizing Parameter Recovery
+
+A key advantage of neural estimation is the ability to quickly conduct inference after training. For example, we can visualize the recovery of parameters:
+
+```julia
+# Extract data from assessment
+df = assessment.df
+
+# Create recovery plots for each parameter
+params = unique(df.parameter)
+p_plots = []
```

Comment:
NeuralEstimators includes similar functionality with `AlgebraOfGraphics.plot(assessment)`. Not that you need to use it but it could be convienent. If decide not to use it, can you make a brief note that it is available?